packages/arb-validator/cmd/rollupServer/rollupServer.go

```diff
@@ -195,13 +195,19 @@ func validateRollupChain() error {
 	}
 
 	ctx := context.Background()
-	manager, err := rollupmanager.CreateManager(ctx, address, true, client, checkpointing.NewRollupCheckpointerImplFactory(
+	manager, err := rollupmanager.CreateManager(
```

Comment:
This api should probably be cleanup up for the common case. Maybe introduce a CreateManager and CreateManagerAdvanced or something like that. We want a common case API that's minimized for inclusion by other users.

Reply:
Did that.